group_project_v2/stage/review.py

```diff
@@ -546,11 +547,21 @@ def load_other_group_feedback(self, request, _suffix=''):
         return webob.response.Response(body=json.dumps(results))
 
     def do_submit_review(self, submissions):
+        user_service = self.runtime.service(self, 'user')
+        reviewer_id = self.anonymous_student_id
+        if 'ta_email' in submissions and submissions['ta_email']:
```

Comment:
This is not the None check I meant. If ta_email is not a valid email, or is of a user that doesn't exists (perhaps due to a typo). Then this will throw and error since a `reviewer_id` of None will be returned by `user_service.get_anonymous_user_id`